src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1348,7 +1346,7 @@ public void MultiDexAndCodeShrinker ([Values ("proguard", "r8")] string linkTool
 		}
 
 		[Test]
-		public void MultiDexR8ConfigWithNoCodeShrinking ([Values (true, false)] bool useConfig)
+		public void MultiDexR8ConfigWithNoCodeShrinking ()
```

Comment:
It occurs to me that this PR fully supersedes the changes from <https://github.com/xamarin/xamarin-android/commit/78dd02167e3b974f246ed1e1f5b386e698c15c0f>, so I think we can delete this test altogether now.

In fact, if it might be OK, maybe this PR could include a full revert of <https://github.com/xamarin/xamarin-android/commit/78dd02167e3b974f246ed1e1f5b386e698c15c0f>? That could be nice because it would remove remove the obsolete extra conditions for `@(ProguardConfigFiles)` in `Xamarin.Android.Common.targets`, and it would also remove the obsolete release note.

Reply:
I added a commit to specifically revert https://github.com/xamarin/xamarin-android/commit/78dd02167e3b974f246ed1e1f5b386e698c15c0f, is it fine that it deletes the release notes?

Reply:
Yup. It's even kind of handy that it removes the release notes too. Thanks!